packages/sdk-react/src/core/queries/usePaymentIntent.ts

```diff
@@ -0,0 +1,11 @@
+import { useMoniteContext } from '../context/MoniteContext';
+
+export const usePaymentIntentById = (payment_intent_id: string) => {
```

Comment:
I agree with the logic of if a function is only used once it should be associated with the use-case, but for this use-case, this can easily introduce repitition when we need this API handling function for another component, shouldn't we introduce all related API functions like this in the current structure for easier reference and update in the future?